plugins/plugins.go

```diff
@@ -22,6 +22,10 @@ type Plugin interface {
 	Stop(ctx context.Context)
 }
 
+// PluginInitFunc is a function that will initialize a plugin given configuration document config
```

Comment:
Nit: this doc string could be a bit more helpful. Something like...


> PluginInitFunc defines the interface for the constructing plugins from configuration. The function will be called with the plugin manager (which provides access to OPA's storage layer, compiler, and service clients) and the configuration for the plugin itself.

Reply:
Changed this.